encrypted-media-respec.html

```diff
@@ -2138,12 +2194,12 @@
                                 <ol>
                                   <li>
                                     <p>
-                                      If the <var>record of key usage</var> does not contain an entry for <var>block key ID</var>, add a new entry for <var>block key ID</var> with <var>first decrypt time</var> set to the current time.
+                                      If the <var>first decrypt time</var> of the object's <var>record of key usage</var> is <code>null</code>, set the <var>first decrypt time</var> to the current time.
                                     </p>
                                   </li>
                                   <li>
                                     <p>
-                                      Set the <var>latest decrypt time</var> of the entry for <var>block key ID</var> in the <var>record of key use</var> to the current time.
+                                      Set the <var>latest decrypt time</var> of the object's <var>record of key usage</var> to the current time.
```

Comment:
nit: You might be able to simplify this using a local variable named _usage_. Then you can set _usage_'s <var>latest decrypt time</var>. Similar in the previous step.